src/mixins/validatable-http-message.coffee

```diff
@@ -138,19 +138,20 @@ class Validatable
     contentType = @headers?['content-type']
     if @isJsonContentType contentType
       try
-        jsonlint.parse @body
+        jph.parse @body
         @validation.body.realType = contentType
       catch error
         message = {
-          message: 'Real body "Content-Type" header is "' +
-            contentType + '" but body is not a parsable JSON.'
+          message: """\
+            Real body 'Content-Type' header is '#{contentType}' \
+            but body is not a parseable JSON:\n#{error.message}\
+          """
           severity: 'error'
         }
-        message.message  = message.message + "\n" + error.message
         @validation.body.results.push message
     else
       try
-        JSON.parse @body
+        jph.parse @body
```

Comment:
In this case we're not handling the error, so I wonder whether using `JSON.parse()` wouldn't be faster 🤔 

OTOH, it could bring inconsistencies in how Gavel parsers JSON and what it considers to be valid or not - not sure whether there are any discrepancies, but [the probability is not low](http://seriot.ch/parsing_json.php). I guess we're better safe using the same implementation everywhere.